gsa/src/web/pages/extras/cvsscalculatorpage.js

```diff
@@ -359,21 +350,388 @@ const CvssCalculator = ({gmp, onInteraction, ...props}) => {
+        <Select
+          items={[
+            {
+              value: 'UNCHANGED',
+              label: _('Unchanged'),
+            },
+            {
+              value: 'CHANGED',
+              label: _('Changed'),
+            },
+          ]}
+          name="scope"
+          value={scope}
+          menuPosition="adjust"
+          onChange={handleMetricsChange}
+        />
+      </FormGroup>
+      <FormGroup title={_('Confidentiality')}>
+        <Select
+          items={[
+            {
+              value: 'NONE',
+              label: _('None'),
+            },
+            {
+              value: 'LOW',
+              label: _('Low'),
+            },
+            {
+              value: 'HIGH',
+              label: _('High'),
+            },
+          ]}
+          name="confidentialityImpact"
+          value={confidentialityImpact}
+          onChange={handleMetricsChange}
+        />
+      </FormGroup>
+      <FormGroup title={_('Integrity')}>
+        <Select
+          items={[
+            {
+              value: 'NONE',
+              label: _('None'),
+            },
+            {
+              value: 'LOW',
+              label: _('Low'),
+            },
+            {
+              value: 'HIGH',
+              label: _('High'),
+            },
+          ]}
+          name="integrityImpact"
+          value={integrityImpact}
+          menuPosition="adjust"
+          onChange={handleMetricsChange}
+        />
+      </FormGroup>
+      <FormGroup title={_('Availability')}>
+        <Select
+          items={[
+            {
+              value: 'NONE',
+              label: _('None'),
+            },
+            {
+              value: 'LOW',
+              label: _('Low'),
+            },
+            {
+              value: 'HIGH',
+              label: _('High'),
+            },
+          ]}
+          name="availabilityImpact"
+          value={availabilityImpact}
+          menuPosition="adjust"
+          onChange={handleMetricsChange}
+        />
+      </FormGroup>
+
+      <h3>{_('From Vector')}:</h3>
+      <FormGroup title={_('CVSS v3.1 Vector')}>
+        <StyledTextField
+          name="userVector"
+          value={userVector}
+          onChange={handleInputChange}
+          onBlur={handleVectorChange}
+          onKeyDown={handleKeyDown}
+        />
+      </FormGroup>
+
+      <h3>{_('Results')}:</h3>
+      <FormGroup title={_('CVSS Base Vector')}>
+        <span>{cvssVector}</span>
+      </FormGroup>
+      <FormGroup title={_('Severity')}>
+        <SeverityBar severity={cvssScore} />
+      </FormGroup>
+    </Layout>
+  );
+};
+
+const CvssCalculator = props => {
+  return (
+    <Layout flex="column">
+      <ToolBarIcons />
+      <Divider margin="10px" flex="row" align={['center', 'start']} grow>
+        <CvssV2Calculator grow="1" props={props} />
+        <CvssV3Calculator grow="1" props={props} />
+      </Divider>
+    </Layout>
+  );
 };
```

Comment:
Just for your info. You can use the short syntax of the array function if it just returns something
```suggestion
const CvssCalculator = props => (
  <Layout flex="column">
    <ToolBarIcons />
    <Divider margin="10px" flex="row" align={['center', 'start']} grow
      <CvssV2Calculator grow="1" props={props} />
      <CvssV3Calculator grow="1" props={props} />
    </Divider>
  </Layout>
);
```